analysis/preprocess/preprocess_data.R

```diff
@@ -0,0 +1,157 @@
+# Load libraries ---------------------------------------------------------------
+
+library(magrittr)
+library(dplyr)
+library(tidyverse)
+library(lubridate)
+library(data.table)
+library(readr)
+
+# Specify command arguments ----------------------------------------------------
+
+args <- commandArgs(trailingOnly=TRUE)
+print(length(args))
+if(length(args)==0){
+  cohort_name <- "vax"
+} else {
+  cohort_name <- args[[1]]
+}
+
+# Get column names -------------------------------------------------------------
+
+all_cols <- fread(paste0("output/input_",cohort_name,".csv.gz"), 
+                  header = TRUE, sep = ",", nrows = 0, 
+                  stringsAsFactors = FALSE) %>%
+  names()
+
+message("Column names found")
+
+# Identify column classes ------------------------------------------------------
```

Comment:
Did the variables "has_follow_up_previous_6months", "was_alive", "has_died" and "vax_jcvi_age_*" get renamed in the end? If so, the following needs to be updated.

Reply:
Done